app/controllers/admin/conference_controller.rb

```diff
@@ -174,8 +174,26 @@ def edit
       end
     end
 
+    def conference_wide_screen
+      # To display sponsors in the conference wide information page
+      @sponsors = @conference.sponsors
+      @program = @conference.program
```

Comment:
You don't need this line. `@program` has already been loaded: https://github.com/openSUSE/osem/blob/master/app/controllers/admin/conference_controller.rb#L4